hasher-matcher-actioner/terraform/actions/.terraform.lock.hcl

```diff
@@ -0,0 +1,20 @@
+# This file is maintained automatically by "terraform init".
```

Comment:
what does this have a sub dir lock? it feels like this is an error but I'm not sure

Reply:
deleted, ran terraform init, and terraform apply. Everything seems fine